snuba/settings_base.py

```diff
@@ -56,8 +56,8 @@
 
 DEFAULT_MAX_BATCH_SIZE = 50000
 DEFAULT_MAX_BATCH_TIME_MS = 2 * 1000
-DEFAULT_QUEUED_MAX_MESSAGE_KBYTES = 10000
-DEFAULT_QUEUED_MIN_MESSAGES = 10000
+DEFAULT_QUEUED_MAX_MESSAGE_KBYTES = None  # use librdkafka default
+DEFAULT_QUEUED_MIN_MESSAGES = None  # use librdkafka default
```

Comment:
why not removing them altogether ? Is it to still be able to make a global change ?

Reply:
Yeah, I think it is still beneficial to be able to change the default value for this configuration without having to specify it in each command line invocation.